### ISSUE TITLE
*Tasks* should not be code-formatted to *Test Cases*

### ISSUE THREAD

**Issue by fabioz** [MarketSquare/robotframework-tidy#279]
It seems that the default is code-formatting `*** Task ***` or even `*** Tasks ***` to `*** Test Cases ***`.

That doesn't seem right to me...

**Comment by bhirsz** [comment]
You're right, it's a bug. I see the problem already here: https://github.com/MarketSquare/robotframework-tidy/blob/69e407eb8efaa06e5c58b29d506891f312812e93/robotidy/transformers/NormalizeSectionHeaderName.py#L38 (I'm actually checking if header is ``task`` but I should make it case insensitive as it will not work with ``Task``).

**Comment by bhirsz** [comment]
@fabioz Released new robotidy as 2.1.1
